find_ndjson.go

```diff
@@ -353,4 +364,13 @@ LOOP:
 		}
 	}
 	latencyTags = append(latencyTags, tag.Insert(metrics.Found, "yes"))
+	yesno := func(yn bool) string {
+		if yn {
+			return "yes"
+		}
+		return "no"
+	}
+
+	latencyTags = append(latencyTags, tag.Insert(metrics.FoundCaskade, yesno(foundCaskade)))
+	latencyTags = append(latencyTags, tag.Insert(metrics.FoundRegular, yesno(foundRegular)))
```

Comment:
why do we have these as two separate tags w/ yes/no values, vs: `{backend: "ipni|cascade"}`

Reply:
Because it's not either / or - both can be true in the same time.

Reply:
The reported metric is for aggregate for all results from all backends (caskade + no-caskade)